prisma/migrations/20240530174040_paid_actions/migration.sql

```diff
@@ -0,0 +1,312 @@
+-- CreateEnum
+CREATE TYPE "InvoiceActionType" AS ENUM ('BUY_CREDITS', 'ITEM_CREATE', 'ITEM_UPDATE', 'ZAP', 'DOWN_ZAP', 'DONATE', 'POLL_VOTE', 'TERRITORY_CREATE', 'TERRITORY_UPDATE', 'TERRITORY_BILLING', 'TERRITORY_UNARCHIVE');
```

Comment:
`BOUNTY_PAYMENT` is included in `ZAP` since it's like a zap, right?

Just want to make sure this wasn't added here on purpose.